src/components/libraryoptionseditor/libraryoptionseditor.template.html

```diff
@@ -205,3 +205,29 @@ <h2>${Lyrics}</h2>
         <div class="fieldDescription checkboxFieldDescription">${SaveLyricsIntoMediaFoldersHelp}</div>
     </div>
 </div>
+
+<div class="audioTagSettingsSection hide">
+    <h2>${LabelAudioTagSettings}</h2>
+    <div class="checkboxContainer checkboxContainer-withDescription advanced">
+        <label>
+            <input type="checkbox" is="emby-checkbox" id="chkPreferNonstandardArtistsTag" />
+            <span>${PreferNonStandardArtistsTag}</span>
+        </label>
+        <div class="fieldDescription checkboxFieldDescription">${PreferNonstandardArtistsTagHelp}</div>
+    </div>
+    <div class="checkboxContainer checkboxContainer-withDescription advanced">
+        <label>
+            <input type="checkbox" is="emby-checkbox" id="chkUseCustomTagDelimiters" />
+            <span>${UseCustomTagDelimiters}</span>
+        </label>
+        <div class="fieldDescription checkboxFieldDescription">${UseCustomTagDelimitersHelp}</div>
+    </div>
+    <div class="inputContainer">
+        <input type="text" is="emby-input" id="customTagDelimitersInput" label="${LabelcustomTagDelimiters}" value="/|;\"/>
```

Comment:
PascalCase in string key for uniformity.
```suggestion
        <input type="text" is="emby-input" id="customTagDelimitersInput" label="${LabelCustomTagDelimiters}" value="/|;\"/>
```